.github/workflows/code-quality.yml

```diff
@@ -20,17 +20,36 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Download wheels from commit OpenAssetIO main
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: build-wheels.yml
+          workflow_conclusion: success
+          name: openassetio-wheels
+          repo: OpenAssetIO/OpenAssetIO
+          path: deps
+
       - name: Install dependencies
         run: |
+          # Use wheels from feature branch for intermediate testing
+          python -m pip install ./deps/openassetio-*cp39*-manylinux_*_x86_64.whl
           python -m pip install -r requirements.txt
-          python -m pip install -r tests/requirements.txt
+          # python -m pip install -r tests/requirements.txt
```

Comment:
[fix] Not keen on reproducing all these `pip install`s. We could comment out the `openassetio` line in `tests/requirements.txt` instead. When we do revert back to using a PyPI release we'd soon notice if we forget to uncomment it.

Reply:
yeah, way smart https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/51/commits/4e474aa986f3ead9d7bca3c18a8fbeecb14686d7